ts/webui/src/components/overview/command/Command1.tsx

```diff
@@ -32,8 +37,8 @@ export const Command1 = (): any => {
             <div>
                 <p className='command'>Training platform</p>
                 <div className='nowrap'>{EXPERIMENT.profile.params.trainingServicePlatform}</div>
-                <p className='lineMargin'>{title}</p>
-                <div className='nowrap'>{builtinName}</div>
+                <p className='lineMargin'>{title.join('/')}</p>
```

Comment:
.join(' / ')

Reply:
seems like `Tuner/Assessor/Advisor` is better than `Tuner / Assessor / Advisor`, It's also similar with the website style.
![image](https://user-images.githubusercontent.com/61399850/99056886-eed30280-25d5-11eb-854c-4ff7a9d0e350.png)